Dockerfile

```diff
@@ -17,7 +17,8 @@ WORKDIR /app
 
 # install dependencies
 COPY poetry.lock pyproject.toml ./
-RUN poetry install --only main
+RUN poetry install --with test,lint,parcing,documents,security,database,main
```

Comment:
we need only main module by defauly, bcs test and lint will be included in future tickets